### ISSUE TITLE
Disable resume timestamp with the VOD.

### ISSUE THREAD

**Issue by rex5207** [justintv/Twitch-API#654]
When I use the embed-video and use following code to initial twitch player
```
var player = new Twitch.Player("{PLAYER_DIV_ID}", options);
```

It will auto resume the timestamp to where I paused this VOD before

How should I disable this function?

Thanks a lot .

**Comment by jwmalara** [comment]
Have you tried seeking to 0 on READY or PLAY? I was looking for a solution to this a while back, but unfortunately wasn't able to find a way of disabling it.

**Comment by rex5207** [comment]
Yes, I try to seek to 0 on READY or PLAY.
But it will still move the timestamp to where I paused soon.

**Comment by jwmalara** [comment]
Yeah, can confirm I'm seeing the same behavior. It would be nice if we had the option to disable resuming or at least an event that's emitted after the player seeks to the resumed position.
